docusaurus/docs/quickstarts/network/local-with-docker.md

```diff
@@ -85,20 +85,38 @@ docker run \
   /openziti/scripts/run-controller.sh
 ```
 
+## Create Edge Router Policies
+For the purposes of quickstart, it is necessary to create an Edge Router Policy and a 

```

Comment:
What do you think of:

OpenZiti requires specific authorization of users and routers (edge router policies), as well as authorization of services and routers  (service edge router policies) . The docker-based quickstart doesn't perform these steps automatically. Run the initialization container one time, after starting the controller as shown

Reply:
Oh, and add links to doc pages for each idea